server/mscalendar/mscalendar.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package mscalendar
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/config"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/store"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/bot"
+)
+
+type MSCalendar interface {
+	Availability
+	Calendar
+	EventResponder
+	Subscriptions
+	Users
+}
+
+// Dependencies contains all API dependencies
+type Dependencies struct {
+	Logger    bot.Logger
+	PluginAPI PluginAPI
+	Poster    bot.Poster
+	Remote    remote.Remote
+	Store     store.Store
+}
+
+type PluginAPI interface {
+	GetMattermostUser(mattermostUserID string) (*model.User, error)
+	GetMattermostUserByUsername(mattermostUsername string) (*model.User, error)
+	GetMattermostUserStatusesByIds(userIDs []string) ([]*model.Status, error)
```

Comment:
Nit, found one outlier
```suggestion
	GetMattermostUserStatusesByIds(mattermostUserIDs []string) ([]*model.Status, error)
```